src/App.tsx

```diff
@@ -1,16 +1,16 @@
-import React from 'react'
-import { Route, Routes } from 'react-router-dom'
-
-import Map from '@pages/Map'
-
-function App() {
-  return (
-    <div>
-      <Routes>
-        <Route path="/" element={<Map />} />
-      </Routes>
-    </div>
-  )
-}
-
-export default App
+import React from 'react'
+import { Route, Routes } from 'react-router-dom'
+
+import Map from '@pages/Map/Map'

```

Comment:
(p4) Map 하위에 Map 이 있는게 어색하네요

`pages/Map/Map.tsx` 을  `pages/Map/index.tsx` 로 옮기면 아래와 같이 깔끔하게 사용할 수 있을 것 같아요

```suggestion
import Map from '@pages/Map'
```